src/ChangePrecision.jl

```diff
@@ -9,7 +9,8 @@ export @changeprecision
 
 const randfuncs = (:rand, :randn, :randexp) # random-number generators
 const matfuncs = (:ones, :zeros, :eye) # functions to construct arrays
-const intfuncs = (:/, :inv, :sqrt, :√, :float) # functions that turn integers into floats
+const intfuncs = (:/, :inv, :sqrt, :√, :float, :sin, :cos, :tan, :asin, :acos, :acosh, :atanh, :log, :log2, :log10,
+                  :lgamma, :log1p, :cbrt, :sinh, :cosh, :tanh, :atan, :asinh, :exp2, :expm1) # functions that turn integers into floats
```

Comment:
Probably `sind` and `cosd` as well, which might actually be commonly called with integer angles.  And also `exp`.

Reply:
Also `:∛`, which is equivalent to a call to `cbrt`.

Reply:
Was just committing this :-) good catch on cbrt. To be on the safe side I systematically added everything (that returned floats) in https://docs.julialang.org/en/release-0.6/manual/mathematical-operations/